### PR TITLE
chore: remove unnecessary default code

### LINKED ISSUE #516: remove code from app component
*SUGGESTION* i think we should remove this from the main app component too

```
  defaultMeaning: number = 42;

  meaningOfLife(meaning?: number) {
    return `The meaning of life is ${meaning || this.defaultMeaning}`;
  }
```

HANS - Agreed, but an empty file is unwelcoming. What would you suggest for a replacement?  IMO a bad joke is better than an empty class. Maybe a comment instead? /** HERE BE DRAGON^H^H^H^H^H^H CODE */ :shrug: 

PAPA - honestly … none of my apps have any code in the app.component.ts. The point of this component is the providers, routing, and the base template for the layout. I’m looking to see what would make sense in there … yes … I love the --tutorial idea you keep bringing up 

MIKE - having that there to ensure that our tests run successfully does add some value for us… but I can see devs not wanting it, but down the road I would LOVE to add an option like --tutorial where they can get some sample code added (with tests)

HANS - --tutorial would be nice. App blueprints could be empty, but it would be nice when `ng g component` to have some code in it.

PAPA - for now, I’d remove it.

cc @hansl @Brocco

### PR DESCRIPTION
**Pull request by Brocco**
Fixes #516

### CODE REVIEW

**Comment by hansl** [comment]
Could you rebase this on top of 565 when it goes in?  I've removed 1-2 things but yours need to go in as well. Great work!

File: addon/ng2/blueprints/component/files/__path__/__name__.component.html [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <p>
```
Comment (filipesilva) [inline addon/ng2/blueprints/component/files/__path__/__name__.component.html]: What about the `<h1>` :(

Comment (filipesilva) [inline addon/ng2/blueprints/component/files/__path__/__name__.component.html]: My bad!!!

Comment (jkuri) [inline addon/ng2/blueprints/component/files/__path__/__name__.component.html]: Should be center

Comment (hansl) [inline addon/ng2/blueprints/component/files/__path__/__name__.component.html]: make it `<marquee>`

**Pull request merged**